guides/v2.1/test/integration/annotations/magentoAppArea.md

```diff
@@ -0,0 +1,114 @@
+---
+group: integration-testing
+version: 2.1
+title: Application Area Annotation in the Integration Testing Framework
+github_link: test/integration/annotations/magentoAppArea.md
+---
+
+Integration testing framework enables you to configure the application area to run tests using the `@magentoAppArea` annotation.
+
+## Format
+
+> Configuring the test environment in scope of the specified application area
+
+```php?start_inline=1
+/**
+ * @magentoAppArea <areaCode>
+ */
+```
+
+## Usage
+
+The annotation is applied following the fallback scheme below:
+
+1. Test annotation
+2. Test case annotation
+3. Default application area, which is `global`
+
+### In a test case
+
+The annotation in a test case enables the specified application area for all tests in the test case.
+If the annotation is specified over a test in the test case, it will overwrite the test case annotation.
+
+> The test case annotation example:
+
+```php?start_inline=1
+/**
+ * @magentoAppArea adminhtml
+ */
+namespace Vendor\Module;
+class ClassToTest extends PHPUnit\Framework\TestCase
+{
+    public function testOne()
+    {
+        //...
+    }
+ 
+    /**
+     * @magentoAppArea frontend
+     */
+    public function testTwo()
+    {
+        //...
+    }
+ 
+    public function testThree()
+    {
+        //...
+    }
+}
+```
+
+In the above example, `testOne` and `testTwo` will be executed in scope of the `adminhtml` application area.
+
+The default value for `@magentoAppArea` is `global`.
+{:.bs-callout .bs-callout-info}
+
+### In a test
+
+The annotation in a test is used to configure the environment in scope of the specified application area for the test.
+Each time you specify a different area, Magento will be reinitialized in the specified scope.
```

Comment:
There is a lot of use of "will be".  It is more clear (and less speculative) when you read "Each time you specify a different area, Magento initializes in the specific scope."

Or something similar. In other words, say what it does, not what it will do. Example: If you press this red button, the rocket launches into space.